### DISCUSSION TITLE
Abnormal readings

### DISCUSSION THREAD

**Discussion by slepax** [klaasnicolaas/home-assistant-glow#308] [Q&A]
I've been using the HASS Glow project for quite some time now to monitor household energy consumption. Thank you for that 🙌

Recently I have been having some strange abnormal readings at the start of the day.
It matches other times where we usually have a spike in energy consumption (morning rush), but the abnormal readings are way out of bound and do not make sense at all.

See in the attached image - the reading actually shows 1.8m W consumed, which is not even possible in our house (single phase, 220v, main switch 80A). And it's not just a single reading, it is several of them over multiple time entries.

So just wondering if anyone has come across something like that?
Other interesting point is that it topped out at 1.8m several times, what's so special about this number?

Other than putting a limit in the code to ignore values over a certain threshold, any other ideas?

![HASS Glow - Power Consumption](https://github.com/klaasnicolaas/home-assistant-glow/assets/13784746/83be758f-6308-4558-bd48-5ea20e3de381)

**Comment by jiikoo** [comment]
Our power measurement worked flawlessly for months until suddenly, large power spikes began appearing in the measurements in the mornings and evenings, sometimes exceeding 100GW. I suspect the issue arose from the recent dismantling of the 3G mobile network in our area, causing the electricity meter to switch to using 2G for transmitting measurement data. Apparently, the 2G transmission interfered with the unprotected wire between the sensor and the ESP. After replacing the unshielded wire with a shielded microphone cable, the problems disappeared.
